js/views/modals/Settings/Moderation.js

```diff
@@ -21,52 +25,51 @@ export default class extends baseVw {
       this.profile.set('modInfo', this.moderator);
     }
 
+    this.currencyList = getTranslatedCurrencies(app.settings.get('language'));
+
     this.listenTo(this.profile, 'sync', () => {
       app.profile.set({
         moderator: this.profile.get('moderator'),
         modInfo: this.profile.get('modInfo').toJSON(),
       });
     });
-
-    // When this bug is fixed and we're saving via PATCH,
-    // the block below could be removed:
-    // https://github.com/OpenBazaar/openbazaar-go/issues/314
-    this.listenTo(app.profile, 'someChange', () => {
-      const profileAttrs = _.omit(app.profile.toJSON(),
-        ['moderator', 'modInfo']);
-      this.profile.set(profileAttrs);
-    });
   }
 
   events() {
     return {
-      'click .js-save': 'save',
+      'click .js-save': 'saveCheck',
+      'change #moderationFeeType': 'changeFeeType',
     };
   }
 
   getFormData() {
     return super.getFormData(this.$formFields);
   }
 
+  saveCheck() {
+    /* if the user isn't already a moderator, the status is true, and the confirmation checkboxes
+       aren't checked, show an error */
+    const blockSave = (!this.profile.get('moderator') &&
+      $('input[name=moderator]:checked').val() === 'true' &&
+      !this.$understandRequirements.prop('checked') && !this.$acceptGuidelines.prop('checked'));
+
+    this.$moderationConfirmError.toggleClass('hide', !blockSave);
+    if (!blockSave) this.save();
+  }
+
   save() {
     const formData = this.getFormData();
     this.profile.set(formData);
 
-    const save = this.profile.save();
-
-    // When server bug is fixed, use the block below to save
-    // instead of the one above.
-    // https://github.com/OpenBazaar/openbazaar-go/issues/314
-
-    // const save = this.profile.save(formData, {
-    //   attrs: formData,
-    //   type: 'PATCH',
-    // });
+    const save = this.profile.save(formData, {
+      attrs: formData,
+      type: 'PATCH',
+    });
 
     if (save) {
       const msg = {
         // msg: app.polyglot.t('settings.generalTab.statusSaving'),
```

Comment:
There's still a bunch of commented out status message translations. Can they be removed?

```
      const msg = {
        // msg: app.polyglot.t('settings.generalTab.statusSaving'),  // <=== Can we remove these? There's a few in this file.
        msg: app.polyglot.t('settings.moderationTab.status.saving'),
        type: 'message',
      };
```